src/mono/browser/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -151,15 +151,19 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, JObject par
                             return false;
                         try
                         {
+                            var url = args["url"]?.ToString();
+                            if (url?.Contains("/_framework/") == true)//it is from dotnet runtime framework
+                            {
+                                context.FrameworkScriptList.Add(args["scriptId"].Value<int>());
+                                return false;
+                            }
+                            if (url?.Equals("") == false)
+                                return false;
                             var callStack = args["stackTrace"]?["callFrames"]?.Value<JArray>();
                             var topFrameFunctionName = callStack?.Count > 0 ? callStack?[0]?["functionName"]?.Value<string>() : null;
                             //skip mono_wasm_fire_debugger_agent_message_with_data_to_pause or mono_wasm_runtime_ready (both of them have debugger; statement)
-                            if (args["url"]?.ToString()?.Equals("") == true && topFrameFunctionName?.StartsWith("mono_wasm_", StringComparison.OrdinalIgnoreCase) == true)
+                            if (topFrameFunctionName?.StartsWith("mono_wasm_", StringComparison.OrdinalIgnoreCase) == true)
```

Comment:
Could we make top level constant out of this `mono_wasm_` prefix ? 
I imagine that one day we may share the JS code with NAOT, which is not Mono. We will probably rename it by then.

cc @maraf

Reply:
I mean in all places.

Reply:
This debugger won't work with NativeAOT so I don't think we need to worry about this at the moment